sql/core/src/test/resources/sql-tests/inputs/pgSQL/aggregates_part2.sql

```diff
@@ -7,207 +7,211 @@
+--   BOOL_AND(b4)     AS "n",
+--   BOOL_AND(NOT b2) AS "f",
+--   BOOL_AND(NOT b3) AS "t"
+-- FROM bool_test;
+
+-- SELECT
+--   EVERY(b1)     AS "f",
+--   EVERY(b2)     AS "t",
+--   EVERY(b3)     AS "f",
+--   EVERY(b4)     AS "n",
+--   EVERY(NOT b2) AS "f",
+--   EVERY(NOT b3) AS "t"
+-- FROM bool_test;
+
+-- SELECT
+--   BOOL_OR(b1)      AS "t",
+--   BOOL_OR(b2)      AS "t",
+--   BOOL_OR(b3)      AS "f",
+--   BOOL_OR(b4)      AS "n",
+--   BOOL_OR(NOT b2)  AS "f",
+--   BOOL_OR(NOT b3)  AS "t"
+-- FROM bool_test;
 
 --
 -- Test cases that should be optimized into indexscans instead of
 -- the generic aggregate implementation.
 --
 
 -- Basic cases
-explain (costs off)
+explain
   select min(unique1) from tenk1;
 select min(unique1) from tenk1;
-explain (costs off)
+explain
   select max(unique1) from tenk1;
 select max(unique1) from tenk1;
-explain (costs off)
+explain
   select max(unique1) from tenk1 where unique1 < 42;
 select max(unique1) from tenk1 where unique1 < 42;
-explain (costs off)
+explain
   select max(unique1) from tenk1 where unique1 > 42;
 select max(unique1) from tenk1 where unique1 > 42;
 
 -- the planner may choose a generic aggregate here if parallel query is
 -- enabled, since that plan will be parallel safe and the "optimized"
 -- plan, which has almost identical cost, will not be.  we want to test
 -- the optimized plan, so temporarily disable parallel query.
-begin;
-set local max_parallel_workers_per_gather = 0;
-explain (costs off)
+-- begin;
+-- set local max_parallel_workers_per_gather = 0;
+explain
   select max(unique1) from tenk1 where unique1 > 42000;
 select max(unique1) from tenk1 where unique1 > 42000;
-rollback;
+-- rollback;
 
 -- multi-column index (uses tenk1_thous_tenthous)
-explain (costs off)
+explain
   select max(tenthous) from tenk1 where thousand = 33;
 select max(tenthous) from tenk1 where thousand = 33;
-explain (costs off)
+explain
   select min(tenthous) from tenk1 where thousand = 33;
 select min(tenthous) from tenk1 where thousand = 33;
 
 -- check parameter propagation into an indexscan subquery
-explain (costs off)
+explain
   select f1, (select min(unique1) from tenk1 where unique1 > f1) AS gt
     from int4_tbl;
 select f1, (select min(unique1) from tenk1 where unique1 > f1) AS gt
   from int4_tbl;
 
 -- check some cases that were handled incorrectly in 8.3.0
-explain (costs off)
+explain
   select distinct max(unique2) from tenk1;
 select distinct max(unique2) from tenk1;
-explain (costs off)
+explain
   select max(unique2) from tenk1 order by 1;
 select max(unique2) from tenk1 order by 1;
-explain (costs off)
+explain
   select max(unique2) from tenk1 order by max(unique2);
 select max(unique2) from tenk1 order by max(unique2);
-explain (costs off)
+explain
   select max(unique2) from tenk1 order by max(unique2)+1;
 select max(unique2) from tenk1 order by max(unique2)+1;
-explain (costs off)
+explain
   select max(unique2), generate_series(1,3) as g from tenk1 order by g desc;
```

Comment:
Our `range(1,3)` can not replace `generate_series(1,3)` here.  How to deal with this case?

Reply:
rewrite the query?

Reply:
Done